modules/tensor_mechanics/src/kernels/ADGravity.C

```diff
@@ -0,0 +1,43 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ADGravity.h"
+
+registerADMooseObject("TensorMechanicsApp", ADGravity);
+
+/**
+ * This kernel defines the residual contribution from a gravitational body force
+ */
+defineADValidParams(
+    ADGravity,
+    ADKernelValue,
+    params.addClassDescription("Apply gravity. Value is in units of acceleration.");
+    params.addParam<bool>("use_displaced_mesh", true, "Displaced mesh defaults to true");
+    params.addParam<Real>("value",
+                          -9.8,
```

Comment:
This is different from `Gravity.C` which has a default of 0. It seems silly to not have a default of earths gravity, so I'll change it here, but leave `Gravity.C` alone for now.

Reply:
Are the default units for simulations like BISONs SI?

Reply:
Yeah, I get your point...but then why have gravity at all? Units are SI in BISON

Reply:
No, I think that SI is absolutely the right default. I was just going to be sad if BISON wasn't in SI 😄

Reply:
I'm late to the party here, but I disagree about having a default here.  This should be a required parameter, rather than having a default of 0 or 9.8. Codes like Bison have standardized unit systems because they have so many models that are unit-system dependent, but MOOSE shouldn't make assumptions about the unit system, not to mention what planet you are on or where on the planet you are.

Reply:
Whoa, he's alive!
No problem @bwspenc, I can change it and Gravity.C while I'm at it in #13134